test/integration/session_restart_test.exs

```diff
@@ -0,0 +1,50 @@
+defmodule ChromicPDF.SessionRestartTest do
+  use ExUnit.Case, async: false
+
+  @ps_cmd :"ps ax | grep -v grep | grep -i Chrom"
+  @wait_ms 500
+
+  defp chrome_pids do
+    @ps_cmd
+    |> :os.cmd()
+    |> to_string()
+    |> String.trim()
+    |> String.split("\n")
+    |> Enum.map(fn x -> Regex.named_captures(~r/[^\d]*(?<pid>[\d]+)/, x)["pid"] end)
```

Comment:
maybe some comment on how you achieved os compatibility here could save later generations of devs :)